sdcm/utils/nemesis_utils/indexes.py

```diff
@@ -85,4 +85,4 @@ def drop_index(session, ks, index_name) -> None:
             db_event=DatabaseLogEvent.DATABASE_ERROR,
             line="Error applying view update"):
         session.execute(f'DROP INDEX {ks}.{index_name}')
-        time.sleep(30)  # errors can happen only within several seconds after index drop #12977
+        time.sleep(180)  # errors can happen only within few minutes after index drop scylladb/#12977
```

Comment:
Doesn't the DbFilter has it's own mechanized to keep ignoring error ? Or it's just the ChangeSeverity  has it ?


I don't think we need artificial sleep

Reply:
I think ChangeSeverity  has it, do you think would be better to replace with it?

Reply:
Or add the support to this one, but I think it's better option then just slepping

Reply:
use `extra_time_to_expiration` it available on `EventsFilter`

Reply:
thanks, fixed.